tests/integration/test_smoke.py

```diff
@@ -12,109 +12,20 @@
 # language governing permissions and limitations under the License.
 import re
 import os
-import random
 import time
+import json
 
 import pytest
 
 from awscli.testutils import aws
 
+INTEG_TESTS_DIR = os.path.dirname(os.path.join(os.path.abspath(__file__)))
```

Comment:
It looks like we don't need `os.path.join` here.

Reply:
Updated in latest revision